packages/sdk/index.js

```diff
@@ -135,7 +135,7 @@ export class SDK extends EventEmitter {
 
     // Announce the drive as a client
     const discovery = this.join(crypto.discoveryKey(key), { server: false, client: true })
-    drive.on('close', () => discovery?.destroy())
+    discovery && drive.once('close', () => discovery?.destroy())
```

Comment:
The optionality question mark on `discovery?` seems unnecessary

Reply:
Good catch thanks.